components/mdxRenderer.tsx

```diff
@@ -3,6 +3,10 @@ import type { FC } from 'react';
 
 import { htmlComponents, mdxComponents } from '@/next.mdx.use.mjs';
 
+declare global {
```

Comment:
Can this be extracted to a `global.d.ts` file?

Reply:
Nope, that doesn’t work. (I just tried again to make sure.) `.d.ts` are really supposed to be build artifacts, and TypeScript treats them as such. This is often misunderstood. You can try it. Just make silly type error in `next-env.d.ts` (`export const foo: number = ''`). TypeScript don’t complain. In fact, the existence of for example `next-env.d.ts` tells TypeScript there’s a file named `next-env.js` in the same folder.

Reply:
I assume these errors are happening, because `next-env.d.ts` forces some specific shenanigans.

Anyhow, PR is good!